microsoft-365/knowledge/topic-experiences-overview.md

```diff
@@ -58,13 +58,13 @@ Your knowledge admins can choose to crawl all SharePoint sites in your tenant fo
 
 ## Roles
 
-When you use Topic experiences in you Microsoft 365 environment, your users will have the following roles:
+When you use Topic experiences in your Microsoft 365 environment, your users will have the following roles:
 
 - Topic viewer: Users who will be able to see topic highlights on SharePoint modern sites that they have at least *Read* access to, and in Microsoft Search. They will be able to select topic highlights to see topic details in topic pages. Topic viewers will be able to provide feedback on how useful a topic is to them.
 
-- Contributors: Users who have rights to edit existing topics or create new ones. Knowledge admins assign contributor permissions to users through the Topic experiences settings in the Microsoft 365 admin center. Note that you can also choose to give all topic viewers the permission to edit and create topics so that they can also contribute to topics that they see.
+- Contributors: Users who have rights to edit existing topics or create new ones. Knowledge admins assign contributor permissions to users through the Topic experiences settings in the Microsoft 365 admin center. Note that you can also choose to give all topic viewers the permission to edit and create topics so that everyone can contribute to topics that they see.
```

Comment:
```suggestion
- Contributors: Users who have rights to edit existing topics or create new ones. Knowledge admins assign contributor permissions to users through the Topic Experiences settings in the Microsoft 365 admin center. Note that you can also choose to give all topic viewers the permission to edit and create topics so that everyone can contribute to topics that they see.
```